### DISCUSSION TITLE
Disable verbose

### DISCUSSION THREAD

**Discussion by rpatapa** [rany2/edge-tts#255] [Q&A]
In command line interface, is there a way to disable comments generated during execution?
Errors and warnings is fine but, can we disable messages like:
**WEBVTT
00:00:01.271 --> 00:00:03.070**

etc? 

Something like a **quiet mode** of execution?

**Comment by WillJDunn** [comment]
The behavior of redirecting standard out via Bash is also strange. edge-tts seems to be writing everything to standard error, even these informational messages.

One would expect that you could get your desired result by simply redirecting standard output to /dev/null (or your OS' equivalent) and leaving standard error unchanged, such as:
```bash
edge-tts --text "Hello, world" --write-media hello.mp3 >/dev/null
```
However, this command still results in messages:
```bash
WEBVTT

00:00:00.100 --> 00:00:01.250
Hello world
```

But if we redirect standard error instead, those informational messages are successfully redirected:
```bash
edge-tts --text "Hello, world" --write-media hello.mp3 2>/dev/null
└ $
```
However, this is inappropriate since it also suppresses genuine error messages.

**Comment by rany2** [comment]
This is now fixed with https://github.com/rany2/edge-tts/pull/329, it will be available in the next release.
